src/Controller/Framework/AwsStorageController.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+namespace App\Controller\Framework;
+
+use App\Command\CommandDispatcherTrait;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\StreamedResponse;
+use League\Flysystem\AwsS3v3\AwsS3Adapter;
+use League\Flysystem\Filesystem;
+use Symfony\Component\HttpFoundation\Request;
+use Aws\Credentials\CredentialProvider;
+use App\Command\Framework\AddFileToAwsCommand;
+use App\Command\Framework\DeleteFileCommand;
+use App\Entity\Framework\LsItem;
+use Symfony\Component\HttpFoundation\JsonResponse;
+
+class AwsStorageController extends AbstractController
```

Comment:
Most of this file is not AWS specific, so the class name and method names should probably be adjusted to remove that.
Maybe move the configuration part into a separate service to remove all AWS-specific code from this file into a symfoy service and allow different storage providers to be configured.